### ISSUE TITLE
Export should generate string for 'menu-icon' (V1)

### ISSUE THREAD

**Issue by JulienMelissas** [WebDevStudios/custom-post-type-ui#210]
Hey - just putting 1.0 through it's paces like you asked me to... If I get some time to fix some issues I find, I will totally submit PRs, but right now I'm just creating issues.

When I did an export of a CPT I got:
```
'menu_icon' => dashicons-businessman,
```
instead of:
```
'menu_icon' => 'dashicons-businessman',
```

**Comment by tw2113** [comment]
Yeah, that'll need to be bugfixed, and I can get it in before early this upcoming week when I'm hoping to try it again.

If nothing is provided, i need to make sure to have it not export anything or export `null`

Thanks for pointing this out Julien.

**Comment by JulienMelissas** [comment]
@tw2113 do you want me to keep submitting issues for everything I find? I'll go crazy if you want, then maybe if I can get this latest job out of the way I can pitch in to get some of these squashed.

**Comment by tw2113** [comment]
If you're finding issues, submit them. If there's enough of them coming from existing sites using CPTUI, then I would absolutely love them to prevent bug reports in the future as well as regressions from the previous version. A lot of rewriting went into this, so there's plenty of chances for missed holes.

**Comment by JulienMelissas** [comment]
Great. Anything I find in using 1.0 I'll make issues for. I'm currently just using it to generate the CPTs and CTs and then exporting and adding to my own plugin for the project, but at least I can test those things for ya.

**Comment by tw2113** [comment]
Fixed in 16a8f908d63be58714cc56307e43c25ae4e50fd5
